src/shared/views/ChangeDetection/Task.js

```diff
@@ -152,7 +152,14 @@ export default class ChangeDetectionTask extends React.PureComponent<
                     />
                 </View>
                 {tutorial && tutorialText !== '' && (
-                    <TutorialBox>{tutorialText}</TutorialBox>
+                    <TutorialBox
```

Comment:
Yes, it's a bit unrelated, but I changed things here as the data format changed. This was to keep `flow` happy, as it otherwise throws an error, which breaks CI.